docs/tutorial.md

```diff
@@ -0,0 +1,103 @@
+
+ This is how it looks at Etherscan:
+
+ ![abi](assets/abi.png)
+
+You can go over contract's functionality in MyEtherWallet, or read the [reference](reference/CurrencyFactory.md) in our docs. The reference also also contains developers comments, and we will improve it as time goes by. But we're here to issuance out Community Currency, so let's proceed.
+
+The function to issue a currency is called, not surprisingly, `createCurrency`. You need to specify four parameters to call it: `name`, `symbol`, `decimals` and `totalSuply`. They all are part of the [ERC20](https://theethereum.wiki/w/index.php/ERC20_Token_Standard) token standard. Yes, all community currencies issued by CLN contracts are ERC20-compliant (and [ERC667](https://github.com/ethereum/EIPs/issues/677) also). Generally, this means that they can be easily added and used by any ERC20 wallet.
+
+- I call my currency `CommunityCurrency`
+- Set the symbol accordingly to `CC`
+- Define decimals to 18 (this is the standard). This means that `CC` token will be devisible to 18 places after the dot.
+- Total suply defines how much `CC` tokens will be issued. Let's define a total supply of one million tokens, 1e6 * 1e18 = 1000000000000000000000000. This means that one million "full" `CC` coins will be created, every token devisible up to 18 places.
+
+
+This is how it looks at MyEtherWallet:
+
+![mew](assets/mew.png)
+
+I click on "WRITE". You don't send ETH to any of CLN contracts, so I set the ammount to send to 0, and the Gas Limit to 200000. Checking again that I'm on the right network I click Yes.
+
+This is my [transaction](https://ropsten.etherscan.io/tx/0x83e96a696110172da2cf7d0afa11bd7415f6737dbbd51d3055ac609cfe12b206). Looking closely into it you can see that:
+
+- The transaction sent from my account ([0x0d4df041db](https://ropsten.etherscan.io/address/0x0d4df041dbef6ffc0e444a4a213774adb0c118c2)) to the `CurrencyFactory`.
+- The internal invoked contract is the new Community Currency. It says that 1000000 `CC` (the total suply) got transfered from `CurrencyFactory` to some [unknown contract](https://ropsten.etherscan.io/address/0xb3f9a85d00fcb75be507da5efc0b91ed221e9bb9).
+- My Community Currency contract address is [0x8611c307F3b88040Aa4E73E8e2c5DB303ca81701](https://ropsten.etherscan.io/address/0x8611c307f3b88040aa4e73e8e2c5db303ca81701), save address of your Community Currency cause we're going to use it soon.
+
+So what happened here?
+
+- `CurrencyFactory` created the CC. As the token creator the `CurrencyFactory` contract was made the owner of all the suply.
```

Comment:
* was made the owner of all the suply --> is now the owner of all the supply

Reply:
I think "was made" is correct, it's some form of past tense.